mm2src/coins/lightning.rs

```diff
@@ -919,6 +957,52 @@ pub async fn open_channel(ctx: MmArc, req: OpenChannelRequest) -> OpenChannelRes
     })
 }
 
+#[derive(Deserialize)]
+pub struct UpdateChannelReq {
```

Comment:
This can be done in a separate refactor PR maybe. Moving all the lightning RPCs now will add lots of changes to this PR that are not really changes but might confuse the reviewers a bit.